backend/coreapp/compilers.py

```diff
@@ -1823,14 +1828,33 @@ def preset_from_name(name: str) -> Optional[Preset]:
     ),
     Preset("Kingdom Hearts", EE_GCC296, "-O2 -G0 -g"),
     # Windows
-    Preset("LEGO Island", MSVC42, "/W3 /GX /O2 /TP"),
-    Preset("Touhou 6 (C)", MSVC70, "/MT /G5 /GS /Od /Oi /Ob1"),
-    Preset("Touhou 6 (C++)", MSVC70, "/MT /EHsc /G5 /GS /Od /Oi /Ob1 /TP"),
+    Preset(
+        "LEGO Island",
+        MSVC42,
+        "/W3 /GX /O2 /TP",
+        libraries=[Library("directx", "5.0")],
```

Comment:
I think it'd be nice if we had the libraries in the global scope so we could refer to them here instead of creating new Library objects which may or may not exist

Reply:
Fixed.